src/main/java/uk/ac/cam/cl/dtg/isaac/api/managers/EventNotificationEmailManager.java

```diff
@@ -135,11 +137,20 @@ public void sendReminderEmails() {
           if (EventStatus.CANCELLED.equals(event.getEventStatus())) {
             continue;
           }
-          String emailKey = String.format("%s@pre", event.getId());
-          // Includes the attended status in case the events team have pre-emptively marked someone as attended.
-          List<BookingStatus> bookingStatuses = Arrays.asList(BookingStatus.CONFIRMED, BookingStatus.ATTENDED);
-          if (pgScheduledEmailManager.commitToSchedulingEmail(emailKey)) {
-            this.sendBookingStatusFilteredEmailForEvent(event, "event_reminder", bookingStatuses);
+          if (event.getDate().after(endOfToday)) {
+            String emailKey = String.format("%s@pre", event.getId());
```

Comment:
This code is copypasted. Can't it be moved to a simple private function that accepts two parameters (email key and template ID)?